src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenLocalFunctionTests.cs

```diff
@@ -5357,14 +5357,46 @@ static void validate(ModuleSymbol module)
                 var attrs1 = localFn1.GetAttributes();
                 Assert.Equal("CompilerGeneratedAttribute", attrs1.Single().AttributeClass.Name);
 
-                // PROTOTYPE(local-function-attributes): consider preventing the lowered local function from containing DynamicAttribute
                 Assert.Equal("DynamicAttribute", localFn1.GetReturnTypeAttributes().Single().AttributeClass.Name);
 
                 var param = localFn1.Parameters.Single();
                 Assert.Equal("DynamicAttribute", param.GetAttributes().Single().AttributeClass.Name);
             }
         }
 
+        [Fact]
+        public void LocalFunctionParamsArray_ParamArrayAttribute()
```

Comment:
>ParamArrayAttribute [](start = 45, length = 19)

There is no `ParamArrayAttribute` in the test. If that's intentional, consider removing `_ParamArrayAttribute` from the test name.